src/components/Column/FocusedCardComponent/FocusedCardComponent.tsx

```diff
@@ -62,7 +62,7 @@ class FocusedCardComponent extends React.Component<
   };
 
   render() {
-    let theme = 'dark';
+    let theme: 'dark' | 'light' = 'dark';
```

Comment:
I guess this should be of type `ActionTheme`, isn't it?

Reply:
Ehm.. you must have had and outdated copy of the code.. :trollface:

Reply:
You don't say ... ;)